guacamole-common/src/main/java/org/apache/guacamole/GuacamoleException.java

```diff
@@ -68,5 +68,15 @@ public GuacamoleException(Throwable cause) {
     public GuacamoleStatus getStatus() {
         return GuacamoleStatus.SERVER_ERROR;
     }
+
+    /**
+     * Returns the numeric HTTP status code associated with this exception.
+     *
+     * @return
+     *     The numeric HTTP status code associated with this exception.
+     */
+    public Integer getHttpStatusCode() {
```

Comment:
This should a primitive `int`, not `Integer`.

Reply:
Changed.